src/core/server/app/router/client.ts

```diff
@@ -120,6 +126,8 @@ const clientHandler = ({
     enableCustomCSS,
     locale,
     config,
+    enableCustomCSSQuery,
+    customCSSURL: req.query.customCSSURL,
```

Comment:
Just a small suggestion!

```suggestion
    customCSSURL: enableCustomCSSQuery ? req.query.customCSSURL : null,
```

Which might simplify things a tad in the template!